.gitmodules

```diff
@@ -9,7 +9,7 @@
 [submodule "ccpp/physics"]
   path = ccpp/physics
   url = https://github.com/NCAR/ccpp-physics
-  branch = main
```

Comment:
@JeffBeck-NOAA Please change the branch back to the main branch. Thanks

Reply:
Fixed! Thank you!